aten/src/ATen/native/cuda/SpectralOps.cu

```diff
@@ -261,9 +261,10 @@ static inline Tensor _run_cufft(
   return output;
 }
 
-// The cuFFT plan cache, defined in CuFFTUtils.h
```

Comment:
This comment wasn't correct. `plan_caches` is not declared in `CuFFTUtils.h` and is only used in this file so made it `static`.